pixeltable/tests/test_component_view.py

```diff
@@ -162,9 +216,9 @@ def run_snapshot_test(self, cl: pt.Client, has_column: bool, has_filter: bool, r
         assert status.num_excs == 0
 
         # create frame view with a computed column
-        args = {'video': video_t.video, 'fps': 1}
+        args = {'video': video_t.video}
         view_t = cl.create_view(
-            view_path, video_t, iterator_class=FrameIterator, iterator_args=args, is_snapshot=False)
+            view_path, video_t, iterator_class=TestIterator, iterator_args=args, is_snapshot=False)
```

Comment:
I assume we're still testing `FrameIterator` somewhere else?

Reply:
Yes, the other tests in test_component_view still run against FrameIterator.